lib/cherrypy/test/test_xmlrpc.py

```diff
@@ -1,13 +1,21 @@
 import sys
 
 import six
-
+# https://docs.openstack.org/developer/bandit/blacklists/blacklist_imports.html#b411-import-xmlrpclib
 try:
-    from xmlrpclib import DateTime, Fault, ProtocolError, ServerProxy
-    from xmlrpclib import SafeTransport
+    from xmlrpclib import (DateTime,  # nosec
+                           Fault,
+                           ProtocolError,
+                           ServerProxy,
+                           SafeTransport)
+    # from defusedxml.xmlrpclib import DateTime, Fault, ProtocolError 
+    # from defusedxml.xmlrpclib import ServerProxy, SafeTransport
```

Comment:
Is this a WIP? You have commented out imports here, which are referred to by your link above.

Reply:
this was unintentional, from a wip